src/main/java/youyihj/zenutils/api/item/CrTFluidHandler.java

```diff
@@ -0,0 +1,45 @@
+package youyihj.zenutils.api.item;
+
+import crafttweaker.annotations.ZenRegister;
+import crafttweaker.api.liquid.ILiquidStack;
+import crafttweaker.api.minecraft.CraftTweakerMC;
+import net.minecraftforge.fluids.FluidStack;
+import net.minecraftforge.fluids.capability.IFluidHandler;
+import stanhebben.zenscript.annotations.IterableSimple;
+import stanhebben.zenscript.annotations.ZenClass;
+import stanhebben.zenscript.annotations.ZenMethod;
+
+import java.util.Objects;
+
+/**
+ * @author various.authors
+ */
+@ZenClass("mods.zenutils.FluidHandler")
+@ZenRegister
+public class CrTFluidHandler {
+    private final IFluidHandler fluidHandler;
+
+    private CrTFluidHandler(IFluidHandler itemHandler) {
+        Objects.requireNonNull(itemHandler);
+        this.fluidHandler = itemHandler;
+    }
+
+    public static CrTFluidHandler of(IFluidHandler fluidHandler) {
+        return fluidHandler == null ? null : new CrTFluidHandler(fluidHandler);
+    }
+
+    @ZenMethod
+    public ILiquidStack drain(int maxDrain, boolean doDrain) {
+        return CraftTweakerMC.getILiquidStack(fluidHandler.drain(maxDrain, doDrain));
+    }
+
+    @ZenMethod
+    public ILiquidStack drain(FluidStack resource, boolean doDrain) {
```

Comment:
change the forge's `FluidStack` to crafttweaker's `ILiquidStack`. And `getTankProperties` method is missing.

Reply:
Ha! I totally just hit this while testing! Thank you!

I'm not sure what getTankProperties is. Can you give me an example of how one might use it in a script? (and then I will do that while I implement it).

Reply:
```zenscript
val handler = world.getFluidHandler(pos, null);
val properties = handler.tankProperties;
val fluid = properties[0].content;
print("The tank contains " ~ fluid.commandString);
```